test/run/test.utils.js

```diff
@@ -35,14 +35,67 @@ describe("lib/utils", function () {
       return;
     }
 
-    // see ./mock-winreg.js
     var expected = "fake\\binary\\path";
+
+    // see ./mock-winreg.js
+    // Only mock keys in HKLM (local machine) hive
+    var winreg = function(options) {
+      this.get = function(_, fn) {
+        if (options.hive === winreg.HKLM) {
+          fn(null, {value: expected});
+        } else {
+          fn("Failed", null);
+        }
+      };
+    };
+    winreg.HKLM = 1;
```

Comment:
Let's use `"HKLM"` for consistency with the library at https://github.com/fresc81/node-winreg/blob/a24b16f3a7a403b82e88a07009b19e8ae7a40361/lib/registry.js#L17

SImilarly below.